modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -746,6 +756,11 @@ neo4j@system> STOP DATABASE db1 WAIT;
 $ bin/neo4j-admin database import incremental --stage=prepare --nodes=N1=../../raw-data/incremental-import/c.csv db1
 ----
 . `build` stage:
++
+During this stage, the import tool imports the data into the database.
```

Comment:
The `build` stage doesn't quite import the data into the db, rather imports it, deduplicates and validates the data into a temporary location.

Since you mention `new increment path` above, perhaps use that same term here.